catroid/src/main/java/org/catrobat/catroid/camera/CameraManager.java

```diff
@@ -543,18 +543,19 @@ public void destroyStage() {
 		}
 	}
 
-	public boolean switchToCameraWithFlash() {
+	public boolean isCameraFlashAvailable() {
+		return (frontCameraInformation.flashAvailable || backCameraInformation.flashAvailable);
+	}
+
+	public void switchToCameraWithFlash() {
 		if (hasCurrentCameraFlash()) {
-			return true;
+			return;
 		}
+
 		if (frontCameraInformation.flashAvailable) {
```

Comment:
Same thing goes for this if-else-construct -> NPE if frontCameraInformation is not initialized (because no front camera exists).